os_migrate/tests/unit/test_network.py

```diff
@@ -1,54 +1,141 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import openstack
 import unittest
 
-from ansible_collections.os_migrate.os_migrate.tests.unit import fixtures
 from ansible_collections.os_migrate.os_migrate.plugins.module_utils \
-    import network
+    import const, network
+
+
+def sdk_network():
```

Comment:
Initially i wondered whether these fixtures will be used from multiple unit tests, but it looks like they will be specific to the Network unit test. So i'm moving them from `fixtures.py` here.